.github/workflows/pull-request-checks.yaml

```diff
@@ -100,3 +100,45 @@ jobs:
         run: make -C regression/verilog test
       - name: Print ccache stats
         run: ccache -s
+
+  # This job takes approximately 15 minutes
+  check-centos8-make-gcc:
+    name: CentOS 8
+    runs-on: ubuntu-20.04
+    container:
+      image: centos:8
+    steps:
+      - name: Install Packages
+        run: |
+          sed -i -e "s|mirrorlist=|#mirrorlist=|g" -e "s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-Linux-*
+          yum -y install make gcc-c++ flex bison git rpmdevtools wget zlib-devel
+          wget --no-verbose https://github.com/ccache/ccache/releases/download/v4.8.3/ccache-4.8.3-linux-x86_64.tar.xz
+          unxz ccache-4.8.3-linux-x86_64.tar.xz
+          tar xf ccache-4.8.3-linux-x86_64.tar 
+          cp ccache-4.8.3-linux-x86_64/ccache /usr/bin/
+          # yum install dnf-plugins-core
+          # yum config-manager --set-enabled powertools
+          # yum install glibc-static libstdc++-static
+      - name: cache for ccache
+        uses: actions/cache@v3
+        with:
+          path: /github/home/.ccache
+          key: ${{ runner.os }}-make-gcc-${{ github.ref }}-${{ github.sha }}-PR
```

Comment:
I'd use `${{ runner.os }}-centos8-make-gcc-${{ github.ref }}-${{ github.sha }}-PR` for consistency.

Reply:
ok